maestro-ios/src/main/java/ios/idb/IdbIOSDevice.kt

```diff
@@ -193,6 +195,18 @@ class IdbIOSDevice(
         }
     }
 
+    override fun clearAppState(id: String): Result<Unit, Throwable> {
```

Comment:
we already have `clearAppData`, which looks pretty similar to me

Reply:
@artem888 you mean in internal repo, right?

Reply:
ah yeah, i mixed it up, never mind